index.bs

```diff
@@ -736,6 +738,15 @@ In a situation when a GPU context executes a graph with a constant or an input i
 
 When an {{MLContext}} is created with {{MLContextOptions}}, the user agent selects and creates the underlying execution device by taking into account the application's [=power preference=] and [=device type=] specified in the {{MLPowerPreference}} and {{MLDeviceType}} options.
 
+## Task Source ## {#programming-model-task-source}
+
+The <dfn>ml task source</dfn> is a [=task source=] to be used for all [=tasks=] related to asynchronous compilation and execution of {{MLGraph}}s.
```

Comment:
"ml task source" seems also to be used for context creation, should it also be mentioned here?

Reply:
Added!